website/docs/table-design/table-types/pk-table/merge-engines/versioned.md

```diff
@@ -5,4 +5,66 @@ sidebar_position: 3
 
 # Versioned Merge Engine
 
-TODO: Fill me #459
\ No newline at end of file
+By specifying `'table.merge-engine' = 'versioned'`, users can update data based on the configured version column. Updates will be carried out when the latest value of the specified field is greater than the stored value. If it is less than or empty, no update will be made.
+
+:::note
+When using `versioned` merge engine, there are the following limits:
+- `UPDATE` and `DELETE` statements are not supported
+- Partial update is not supported
+  :::
+
+## Versioned Merge Column
+
+:::note
+The versioned merge column supports the following data types.
+- INT
+- BIGINT
+- TIMESTAMP
+- TIMESTAMP(p)
+- TIMESTAMP_LTZ
+- TIMESTAMP_LTZ(p)
+  :::
+
+example:
+```sql title="versioned"
+
+create table merge_engine_with_version (
```

Comment:
Could you please change SQL keyword to uppercase to align the style of first-row merge engine page?